src/Nelmio/Alice/Fixtures/Parser/Methods/Base.php

```diff
@@ -102,17 +102,18 @@ protected function processIncludes($data, $filename)
     /**
      * @param array $data
      * @param array $includeData
+     * @return array
      */
     protected function mergeIncludeData($data, $includeData)
     {
-        foreach ($includeData as $class => $fixtures) {
-            if (isset($data[$class])) {
-                $data[$class] = array_merge($fixtures, $data[$class]);
+        foreach ($data as $class => $fixtures) {
+            if (isset($includeData[$class])) {
+                $includeData[$class] = array_merge($includeData[$class], $fixtures);
             } else {
-                $data[$class] = $fixtures;
+                $includeData[$class] = $fixtures;
             }
         }
 
-        return $data;
+        return $includeData;
```

Comment:
To me it's just that in this case if the name is not clear enough it should be changed in the signature :)

That being said I'm saying that because I find this line useless that's it. If you or @tshelburne find it more clear to do it this way I'm fine with it.